spec/models/like_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] Kindly Add instructions in your README file on how to run the `specs` test
- [ ] And fix all these failures tests. check here :point_down: 
![image](https://user-images.githubusercontent.com/65068771/154231728-e60ff836-43c5-41c9-a226-371d8edc4d07.png)